Tests/Fixtures/TestProject/project-xcode-12.yml

```diff
@@ -0,0 +1,125 @@
+# NOTE: when Xcode 12 goes GM and the Xcode 11 CI can be dropped,
+# this file can be merged into project.yml. As-is, it generates
+# targets that Xcode 11 doesn't understand or know how to build.
```

Comment:
Could we instead have this file `include:` the normal one, and in here just have `App_Clip:` target?

Reply:
Well, and a version of the app that includes the clip.

Reply:
Done